src/Sylius/Bundle/CoreBundle/Fixture/Factory/ProductExampleFactory.php

```diff
@@ -391,4 +391,17 @@ private function getRandomValueForProductAttribute(ProductAttributeInterface $pr
                 throw new \BadMethodCallException();
         }
     }
+
+    private function generateProductVariantName(ProductVariantInterface $variant): string
+    {
+        $variantOptions = $variant->getOptionValues();
+
+        $variantName = '';
+
+        foreach ($variantOptions as $variantOption) {
+            $variantName .= sprintf('%s ', $variantOption->getValue());
+        }
```

Comment:
We could probably use an [array_reduce](https://www.php.net/manual/en/function.array-reduce.php) function for that?